neuralmonkey/processors/speech.py

```diff
@@ -0,0 +1,42 @@
+from typing import Callable
+
+import numpy as np
+from python_speech_features import mfcc, fbank, logfbank, ssc, delta
+
+from neuralmonkey.readers.audio_reader import Audio
+
+
+FEATURE_TYPES = {f.__name__: f for f in [mfcc, fbank, logfbank, ssc]}
+
+
+# pylint: disable=invalid-name
+def SpeechFeaturesPreprocessor(feature_type='mfcc', delta_order=2,
+                               delta_window=2, **kwargs) -> Callable:
+    """Calculate speech features.
+
+    By default, compute 13 MFCCs + delta + acceleration, i.e. 39 coefficients.
```

Comment:
Možná je to, protože se v téhle oblasti vůbec neorientuju, a ostatním je to zřejmé, ale asi bych jako čtenář dokumentace ocenil, kdyby tu bylo napsáno, že bere ten signál po okýnkách velikosti `delta_window` s posunem `delta_order` a na okýnka uplatní nějakou funkci, ze které vyleze něco, co je najednou ve frekvenční doméně. A jestli nemám pravdu, tak o to víc by to tu mělo být napsáno :-)